src/PKSim.Core/Services/IndividualEnzymeFactory.cs

```diff
@@ -27,6 +27,8 @@ public class IndividualEnzymeFactory : IndividualProteinFactory<IndividualEnzyme
       public IndividualEnzyme AddUndefinedLiverTo(Individual individual)
       {
          var undefinedLiver = CreateMolecule(CoreConstants.Molecule.UndefinedLiver);
+         AddOntogenyParameterTo(undefinedLiver);
+
```

Comment:
Why is it required now?

Reply:
It's not added by default in the expression profile. However, when creating our undefined molecule (Undefine liver and undefined transporter, we probably need it..at least to make sure that the formula references are working). Those molecules are added directly in the individual, bypassing completely the expression profile stuff.

Reply:
ok...

Reply:
we'll have to double check that this works as expected for those "on the fly" generated enzymes and transporters